### DISCUSSION TITLE
Can I use this vendure to create montly subscription for orders ?

### DISCUSSION THREAD

**Discussion by lakshmajee** [vendurehq/vendure#1817] [Q&A]
I have a small eCommerce store that has a feature where the order can be scheduled at monthly or weekly intervals. Is there any way for me to add that feature into vendure (actually on top of vendure within my project) ?

Example: While placing an order customers can choose for how many days they want goods on daily basis. (Basically mine is a rental platform)

**Comment by michaelbromley** [comment]
Yes, you can do this by defining custom fields on the OrderLine entity, which allows you to store subscription data with each order line. https://www.vendure.io/docs/developer-guide/customizing-models/

**Reply by lakshmajee** [reply]
Will have a look at it
